src/Easy.cc

```diff
@@ -1612,6 +1612,38 @@ NAN_METHOD(Easy::SetOpt) {
 
         break;
     }
+
+    // check if option is a blob, and the value is correct
+  } else if ((optionId = IsInsideCurlConstantStruct(curlOptionBlob, opt))) {
+#if NODE_LIBCURL_VER_GE(7, 71, 0)
+    if (value->IsNull()) {
+      setOptRetCode = curl_easy_setopt(obj->ch, static_cast<CURLoption>(optionId), NULL);
+    } else if (value->IsString()) {
+      Nan::Utf8String value(value);
+
+      size_t length = static_cast<size_t>(value.length());
+
+      struct curl_blob blob;
+      blob.data = *value;
+      blob.len = length;
+      blob.flags = CURL_BLOB_COPY;
+
+      setOptRetCode = curl_easy_setopt(obj->ch, static_cast<CURLoption>(optionId), &blob);
```

Comment:
```suggestion
      Nan::Utf8String utf8StringValue(value);

      size_t length = static_cast<size_t>(utf8StringValue.length());

      struct curl_blob blob;
      blob.data = *utf8StringValue;
      blob.len = length;
      blob.flags = CURL_BLOB_COPY;

      setOptRetCode = curl_easy_setopt(obj->ch, static_cast<CURLoption>(optionId), &blob);
```

Just to make it more clear what `value` is and to not use the same variable name that was declared in the outer scope

Reply:
Done.